data/avgi/avgi 1 introduction.txt

```diff
@@ -0,0 +1,875 @@
+			`	"They had to come from sonewhere," you realize.`
+			`	"Exactly. And if they came into Avgi space from somewhere, there might be a way out too. Either way, I think we should pay my friend a visit after you wrap up this mission. Might as well head to the spaceport and see what they want."`
+
+	on decline
+		conversation
+			`	"Disappointment. I see. I will ensure you are not troubled by requests for aid in the future." He flies away, vanishing quickly.`
+
+
+mission "Avgi: Scout Rescue"
+	name "Scout Search and Rescue"
+	description "Find and evacuate the crew of the <npc>."
+	source "Weledos"
+	passengers 12
+	destination "Peripheria"
+	to offer
+		has "Avgi: Defend Ensemble 2: done"
+	to fail
+		has "avgi: warship intro"
+	on offer
+		conversation
+			`	You follow Kokkino's instructions to a prefab building in the spaceport, which looks to be some kind of command center. Kokkino is waiting for you by a table in the middle, which lights up, projecting a map of Avgi space in the air with some colored markings you can't read.`
+			`	"Explanation. We send scouting teams and patrols into the space controlled by the Aberrant regularly. Every expedition beyond Evanescence is essentially behind enemy lines, cut off from resupply and in many cases communication. As you might imagine, such adventures are fraught with danger, and it is unfortunately not unheard of for a scout ship to be lost."`
+			choice
+				`	"Yes, I've found that exploration is often dangerous."`
+					goto exploration
+				`	"Sounds like you should build stronger scout ships."`
+			`	"Agreement. The Polari are a fine line of vessels, but not one made for the environment of war. We have experimented with using the Prismaios as an intelligence platform in more contested environments, but it lacks the versatility and drone bays that has made the Polari so venerable in the first place."`
+			label exploration
+			`	He flicks his wings. "Regret. It is a sad truth, but those on the forefront of expanding our knowledge are often those who face the most danger from the unknown."`
+			`	"But, to the point. A scout team, aboard the Polari-class scout 'Silver Albedo' was sent to survey the Twirl system two weeks ago, but has missed its check in by multiple days. To reach that star cluster and return would normally require a trip of weeks, all thorough perilous territory, and I am ashamed to say that we could be forced to consider writing off the scouts as lost. But I have heard that the visiting human, Captain <last>, is in possession of a new kind of faster-than-light drive, one that can travel without the need for starways. Tell me, is this true, question?"`
+			choice
+				`	"Yes, I have a Jump Drive. It can bypass hyperlanes entirely."`
+			`	"Excitement. Then, the trip could be shortened to mere days, giving us the possibility of mountin a rescue. Are you willing to assist us with this, Captain <last>, question?"`
```

Comment:
```suggestion
			`	"Excitement. Then, the trip could be shortened to mere days, giving us the possibility of mounting a rescue. Are you willing to assist us with this, Captain <last>, question?"`
```
Spotted this while reading the files to get the general vibe of Avgi speech patterns